src/main/java/xbot/common/injection/electrical_contract/CANMotorControllerOutputConfig.java

```diff
@@ -0,0 +1,38 @@
+package xbot.common.injection.electrical_contract;
+
+import edu.wpi.first.units.measure.Current;
+
+import static edu.wpi.first.units.Units.Amps;
+
+public class CANMotorControllerOutputConfig {
+    public enum InversionType {
+        Normal,
+        Inverted
+    }
+
+    public enum NeutralMode {
+        Brake,
+        Coast
+    }
+
+    public InversionType inversionType = InversionType.Normal;
+
+    public NeutralMode neutralMode = NeutralMode.Coast;
+
+    public Current statorCurrentLimit = Amps.of(80);
```

Comment:
⭐ suggest pulling out this `80` magic number into a named const `DEFAULT_CURRENT_LIMIT`